examples/NationalInstruments/highlights/example-bars.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Flot Examples: Range Cursors</title>
```

Comment:
All of these examples need updated titles.

Reply:
Done

Reply:
Sorry, I meant the <title> elements.

Reply:
Fixed